akka-stream/src/main/scala/akka/stream/scaladsl/Graph.scala

```diff
@@ -960,8 +960,8 @@ object ZipWithN {
 class ZipWithN[A, O](zipper: immutable.Seq[A] ⇒ O)(n: Int) extends GraphStage[UniformFanInShape[A, O]] {
   override def initialAttributes = DefaultAttributes.zipWithN
   override val shape = new UniformFanInShape[A, O](n)
-  def out = shape.out
-  val inSeq = shape.inSeq
+  def out: Outlet[O] = shape.out
+  val inSeq: immutable.IndexedSeq[Inlet[A]] = shape.inlets.toIndexedSeq
```

Comment:
needed for binary compatibility since `inSeq` is public. I think `toIndexedSeq` is preferable to `asInstanceOf`.